### PR TITLE
[FSDE-12327] Updated steps component vertical design for Portal Style Guide v2.2

### PR DESCRIPTION
**Pull request by jllr**
https://centurylink.sharepoint.com/sites/ILBrandCenter/DocumentsBrand/Forms/AllItems.aspx?id=%2Fsites%2FILBrandCenter%2FDocumentsBrand%2FPortal%2DStyle%2DGuide%2Epdf&parent=%2Fsites%2FILBrandCenter%2FDocumentsBrand

https://xd.adobe.com/view/79ce720b-3477-429c-9fae-5836af3a7d3a-2c9d/screen/2a92547d-eadb-467d-8d71-d565f20b7e2a/specs/

https://ctl.atlassian.net/browse/FSDE-12327

### CODE REVIEW

File: src/website/views/components/steps/_examples.pug [hunk 1]
```diff
@@ -302,6 +302,257 @@ p.-text
...
+                      p Completed item 
+                        span.-text--2xs  | 17 Feb 2021
+                      button.chi-button.-icon.-xs.-flat.-p--0.-ml--1(id="example__help-button-2" aria-label="Help" data-target="#example__help-popover-2")
+                        i.chi-icon.icon-circle-info-outline.-icon--dark.-mr--0(aria-hidden="true")
+                      section.chi-popover.chi-popover--top(id="example__help-popover-2" aria-modal="true" role="dialog")
+                        .chi-popover__content
+                          p.chi-popover__text
+                            | Helpful information goes here.
+                    .chi-steps__subitem.-completed.-position--relative
+                      i.chi-icon.icon-arrow-right.-sm.-icon--warning.-position--absolute(aria-hidden="true", style="left:-48px;")
```
Comment (mattnickles) [inline src/website/views/components/steps/_examples.pug:354]: Can we create a class for this and remove the inline style?

File: src/website/views/components/steps/_examples.pug [hunk 2]
```diff
@@ -302,6 +302,257 @@ p.-text
...
+                        i.chi-icon.icon-warning.-xs.-icon--warning(aria-hidden="true")
+                        p Pending item 
+                          span.-text--2xs.-text--navy  | 17 Feb 2021
+                        button.chi-button.-icon.-xs.-flat.-p--0.-ml--1(id="example__help-button-4" aria-label="Help" data-target="#example__help-popover-4")
+                          i.chi-icon.icon-circle-info-outline.-icon--dark.-mr--0(aria-hidden="true")
+                        section.chi-popover.chi-popover--top(id="example__help-popover-4" aria-modal="true" role="dialog")
+                          .chi-popover__content
+                            p.chi-popover__text
+                              | Helpful information goes here.
+                      button.chi-button.-xs(style="margin-left: 1.125rem;") Schedule
```
Comment (mattnickles) [inline src/website/views/components/steps/_examples.pug:381]: Can we create an action container (e.g. `chi-steps__actions`) to store buttons or links related to the step items and then create a css rule that targets action containers within `chi-steps__subitem` and apply the margin-left there versus defining inline?

File: src/website/views/components/steps/_examples.pug [hunk 3]
```diff
@@ -302,6 +302,257 @@ p.-text
...
+                          span.-text--2xs.-text--navy  | 17 Feb 2021
+                        button.chi-button.-icon.-xs.-flat.-p--0.-ml--1(id="example__help-button-5" aria-label="Help" data-target="#example__help-popover-5")
+                          i.chi-icon.icon-circle-info-outline.-icon--dark.-mr--0(aria-hidden="true")
+                        section.chi-popover.chi-popover--top(id="example__help-popover-5" aria-modal="true" role="dialog")
+                          .chi-popover__content
+                            p.chi-popover__text
+                              | Helpful information goes here.
+                      button.chi-button.-xs Reschedule
+    .example-tabs.-pl--2
+      ul.chi-tabs#example-vertical
```
Comment (mattnickles) [inline src/website/views/components/steps/_examples.pug:400]: Let's fix this duplicate ID

```suggestion
      ul.chi-tabs#example-order-progress
```

File: src/website/views/components/steps/_examples.pug [hunk 4]
```diff
@@ -302,6 +302,257 @@ p.-text
...
+          ) Web Component
+        li.-active
+          a(
+            href=`#html-vertical`
+            role="tab"
+            aria-selected="true"
+            aria-controls="html-vertical"
+          ) HTML Blueprint
+    .chi-tabs-panel#webcomponent-vertical
+    .chi-tabs-panel.-active#html-vertical
```
Comment (mattnickles) [inline src/website/views/components/steps/_examples.pug:416]:
```suggestion
        li.-disabled
          a(
            href=`#webcomponent-order-progress`
            role="tab"
            aria-selected="false"
            aria-controls="webcomponent-order-progress"
          ) Web Component
        li.-active
          a(
            href=`#html-order-progress`
            role="tab"
            aria-selected="true"
            aria-controls="html-order-progress"
          ) HTML Blueprint
    .chi-tabs-panel#webcomponent-order-progress
    .chi-tabs-panel.-active#html-order-progress
```

**Review by mattnickles — APPROVED** [review]

**Review by janibolkvadze — APPROVED** [review]

**Pull request merged**